REST_API_DOCS.md

```
@@ -0,0 +1,1335 @@
+
+---
+
+## Likes (좋아요)
+
+### 좋아요 삭제
+
+**DELETE** `/posts/postId}/likes/me`
+
+특정 게시글에 등록했던 좋아요를 취소(삭제)합니다.
+
+**Request Headers**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)